jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -73,6 +73,15 @@ def should_export_dbias(bias_shape, query_shape, layout) -> bool:
     _, _, q_N, _ = query_shape
   return b_B == 1 and b_N == q_N
 
+def get_large_negative_number(dtype):
+  # temp WAR as cuDNN has a bug for subtraction between two large negative value
+  if dtype == jnp.bfloat16:
+    return jnp.asarray(-2 << 40, dtype=dtype)
```

Comment:
Can we use -.7 * finfo.maxn as the previous version of this function did?

Reply:
I tried -.7 * finfo.maxn and it has the nan issue and cuDNN team recommended the -2 << 40 as WAR for now.